portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,27 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style="font-family:Georgia;background-color:#937691;">  <!-- Light purple-->
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1 style="background-color:#023C40;" >👩🏽‍💻 Fatima Z. Ali </h1> <!-- Dark green-->
+        <img src="/images/FatimaAli.jpeg" alt="Headshot of a woman (Fatima) in a jacket" style="width:300px;height:300px" class="center">
+        
+        <br><h2 style="background-color:#023C40">👋🏽 Hello! // ¡Hola! // Salaam! </h2>
+        
+        <p>My name is Fatima, and I’m a rising junior at UC Irvine studying Computer Science. 
+        I'm interested in Software Engineering and Data Science, and I'm currently living in the Bay Area. </p>
+        <p>I love to build <a href="https://www.facebook.com/groups/minterns/" title="Check out Minterns!">inclusive 
```

Comment:
TIL about the title attribute. 

In this case I don't think it's necessary tho, since the link text provides sufficient context.